tests/js/server/aql/aql-geo.inc

```diff
@@ -216,916 +105,885 @@
+      }
+      let i = 0;
+      let j = 0;
+      let count = 0;
+      while (i < a.length && j < b.length) {
+        if (a[i] < b[j]) {
+          good = false;
+          msg += "Found key " + a[i] + " in '" + namea + "' but not in '"
+            + nameb + "'\n";
+          ++i;
+        } else if (a[i] > b[j]) {
+          good = false;
+          msg += "Found key " + b[j] + " in '" + nameb + "' but not in '"
+            + namea + "'\n";
+          ++j;
+        } else {
+          ++i;
+          ++j;
+        }
+        if (++count >= 10) {
+          return {good, msg};
+        }
+      }
+      while (i < a.length) {
+        good = false;
+        msg += "Found key " + a[i] + " in '" + namea + "' but not in '"
+          + nameb + "'\n";
+        ++i;
+        if (++count >= 10) {
+          return {good, msg};
+        }
+      }
+      while (j < b.length) {
+        good = false;
+        msg += "Found key " + b[j] + " in '" + nameb + "' but not in '"
+          + namea + "'\n";
+        ++j;
+        if (++count >= 10) {
+          return {good, msg};
+        }
+      }
+      return {good, msg};
+    }
 
-    tearDownAll: function () {
-      db._dropView(viewName);
-      db._drop(collWithoutIndex);
-      db._drop(collWithIndex);
-      db._drop(collWithView);
-      let analyzers = require("@arangodb/analyzers");
-      analyzers.remove("geo_json");
-    },
+    function compare(query, queryView) {
+      let fullScanQuery = `FOR d IN @@coll ${query} RETURN d._key`;
+      let viewQuery = `FOR d IN ${viewName} ${queryView} RETURN d._key`;
+      let invertedIndexQuery = `FOR d IN ${collWithIndex} OPTIONS {indexHint: "inverted", forceIndexHint: true, waitForSync: true} ${query} RETURN d._key`;
 
-////////////////////////////////////////////////////////////////////////////////
-/// @brief tear down
-////////////////////////////////////////////////////////////////////////////////
+      let resFullScanNoIndexes = getQueryResults(fullScanQuery, {"@coll": collWithoutIndex}).sort();
+      let resFullScanWithIndexes = getQueryResults(fullScanQuery, {"@coll": collWithIndex}).sort();
+      let resView = getQueryResults(viewQuery, {}).sort();
+      let resInvertedIndex = getQueryResults(invertedIndexQuery, {}).sort();
 
-    tearDown: function () {
-      truncateAll();
-    },
+      let cmpResFullScanWithIndexes = compareKeyLists("without index", resFullScanNoIndexes, "with index", resFullScanWithIndexes);
+      let cmpResView = compareKeyLists("without index", resFullScanNoIndexes, "with view", resView);
+      let cmpResInvertedIndex = compareKeyLists("without index", resFullScanNoIndexes, "with inverted index", resInvertedIndex);
 
-////////////////////////////////////////////////////////////////////////////////
-/// @brief test single point
-////////////////////////////////////////////////////////////////////////////////
+      // PLEASE UNCOMMENT LINES BELOW AFTER FIXING https://arangodb.atlassian.net/browse/BTS-1184
```

Comment:
https://github.com/arangodb/arangodb/pull/18218

Reply:
Please uncomment this test, it anyway will be conflict between your PR and @cpjulia